### ISSUE TITLE
Could the author share the training setup for the Ref-Youtube-VOS datset please?

### ISSUE THREAD

**Issue by haliphinx** [heshuting555/DsHmp#16]
Thanks for the great work, the method achieves attractive result on the Ref-Youtube-VOS benchmark.

Could you please share the config file and the training script for the Ref-Youtube-VOS please?

**Comment by Sjunshu** [comment]
> 感谢出色的工作，该方法在 Ref-Youtube-VOS 基准测试中取得了有吸引力的结果。
> 
> 您能否分享 Ref-Youtube-VOS 的配置文件和训练脚本？

Hello, have you solved this problem?

**Comment by haliphinx** [comment]
@Sjunshu Unfortunately, no. There are some more recent methods like '[SAMWISE](https://github.com/ClaudiaCuttano/SAMWISE)' which achieve higher accuracy. So I have moved forward.
